packages/core/src/components/toggle/toggle.tsx

```diff
@@ -64,6 +69,10 @@ export class TdsToggle {
     });
   };
 
+  componentWillLoad() {
+    this.labelSlot = this.host.querySelector("[slot='label']");
+  }
+
```

Comment:
I would also like to see a warning triggered if the tdsAriaLabel is not set by the user.

```suggestion
  connectedCallback() {
    if (!this.tdsAriaLabel) {
      console.warn('Tegel Toggle component: tdsAriaLabel prop is missing');
    }
  }
```